code-challenges.test.js

```diff
@@ -52,7 +115,60 @@ const hand3 = [5, 5, 5, 5, 4]
 // Expected output: false
 const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
+    it('determines whether or not the array is a "full house"', () => {
+        expect(paintedLadies(hand1)).toEqual(true)
+        expect(paintedLadies(hand2)).toEqual(false)
+        expect(paintedLadies(hand3)).toEqual(false)
+        expect(paintedLadies(hand4)).toEqual(true)
+        
+    })
+})
+
+// Output: ReferenceError: paintedLadies is not defined
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// Name: paintedLadies
+// Input: array
+// Output: boolean value
+// Process: Create a function that takes in a array an checks if there a multiples of two and three using .includes()
+// #### Edit: I had a few approaches for this challenge. All of them failed, I'm calling it on this one....
```

Comment:
This is a tough one! Make sure to pay attention during assessment review to see an approach to this one!